storage/storage.go

```diff
@@ -209,10 +254,19 @@ func (storage DBStorage) WriteReportForCluster(
 	report types.ClusterReport,
 	lastCheckedTime time.Time,
 ) error {
-	statement, err := storage.connection.Prepare(
-		`INSERT OR REPLACE INTO report(org_id, cluster, report, reported_at, last_checked_at) 
-		 VALUES ($1, $2, $3, $4, $5)`,
-	)
+	var query string
+
+	switch storage.dbDriverType {
+	case DBDriverSQLite:
+		query = `INSERT OR REPLACE INTO report(org_id, cluster, report, reported_at, last_checked_at) 
+		 VALUES ($1, $2, $3, $4, $5)`
+	default:
+		query = `INSERT INTO report(org_id, cluster, report, reported_at, last_checked_at)
+		 VALUES ($1, $2, $3, $4, $5)
+		 ON CONFLICT (org_id, cluster) 
+		 DO UPDATE SET report = $3, reported_at = $4, last_checked_at = $5`
+	}
```

Comment:
unfortunately UPSERT is not defined+supported in SQL standard ;/ welcome to 2020

Reply:
I mean - your solution is correct ATM (unless we switch to Squirel)

Reply:
Is there a standard? We can move it to `case postgres` and throw an error in default case

Reply:
done